src/platform_impl/macos/event_loop.rs

```diff
@@ -50,6 +65,16 @@ impl<T: 'static> EventLoopWindowTarget<T> {
 
 pub struct EventLoop<T: 'static> {
     window_target: Rc<RootWindowTarget<T>>,
+
+    /// We make sure that the callback closure is dropped during a panic
+    /// by making the event loop own it.
+    ///
+    /// Every other reference should be a Weak reference which is only upgraded
+    /// into a strong reference in order to call the callback but then the
+    /// strong reference should be dropped as soon as possible.
+    _callback:
```

Comment:
What's the motive for prefixing this with an underscore? It looks like this field is accessed below.

Reply:
The nightly compiler emits a warning, because this is never read. https://github.com/rust-windowing/winit/pull/1853/checks?check_run_id=1892520990#step:9:63

Reply:
Ah, gotcha! Thanks for improving my understanding of the rules around that.